recipes/xorg/all/conanfile.py

```diff
@@ -42,22 +42,22 @@ def system_requirements(self):
             if tools.os_info.with_apt:
                 packages = ["xorg-dev", "libx11-xcb-dev", "libxcb-render0-dev", "libxcb-render-util0-dev", "libxcb-xkb-dev",
                             "libxcb-icccm4-dev", "libxcb-image0-dev", "libxcb-keysyms1-dev", "libxcb-randr0-dev", "libxcb-shape0-dev",
-                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data"]
+                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data", "libxcb0-util-dev" if tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version <= "14" else "libxcb-util-dev"]
```

Comment:
```suggestion
                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data", "libxcb0-util-dev" if tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version < "15" else "libxcb-util-dev"]
```
sorry, trusty is actually 14.04, which is not <= 14 ! This time, I tested it :)

Reply:
well, don't commit this yet : `dpkg-query: no packages found matching libxcb0-util-dev`

Reply:
Thanks! By the way would this work on 13 and lower or is that too old for us to care?

Reply:
I have no idea, but this distribution is not part of CCI test configurations, so there are most probably a LOT of other problems on all recipes.